website/src/schema-markup/faq.json

```
@@ -156,34 +156,66 @@
         },
         {
             "@type": "Question",
-            "name": "How to use targeting rules in ConfigCat based on sensitive data?",
+            "name": "How can I be sure that my data is safe at ConfigCat?",
             "acceptedAnswer": {
                 "@type": "Answer",
-                "text": "If you want to use targeting rules based on email address, phone number, or other sensitive data, you can use the Confidential text comparators: https://configcat.com/docs/advanced/targeting/#confidential-text-comparators."
+                "text": "The feature flag evaluation is done on the client-side in the ConfigCat SDK. So if you are using the SDK, you can be sure that your data will never leave your system. See ConfigCat's architecture explained here: https://configcat.com/architecture/."
             }
         },
         {
             "@type": "Question",
-            "name": "Are ConfigCat percentage targeting options sticky?",
+            "name": "Can I sign a data processing agreement with ConfigCat?",
             "acceptedAnswer": {
                 "@type": "Answer",
-                "text": "Yes. The percentage-based targeting is sticky by design and consistent across all SDKs.  Also, consider the following:  - All SDKs evaluate the rules in the exact same way. (10% is the same 10% in all SDKs)  - The percentage rules are sticky by feature flag. (10% is a different 10% for each feature flag)  More on stickiness and consistency: https://configcat.com/docs/advanced/targeting/#stickiness--consistency"
+                "text": "Yes, ConfigCat have no trouble signing a data processing agreement (DPA)"
             }
         },
         {
             "@type": "Question",
-            "name": "How can I be sure that my data is safe at ConfigCat?",
+            "name": "Which data is processed and stored from browsers accessing ConfigCat's CDN to pull feature toggles?",
```

```suggestion
            "name": "What data is stored or processed from browsers accessing ConfigCat's CDN when pulling feature toggles?",
```
 Please update the original